CONTRIBUTING.md

```diff
@@ -20,9 +20,22 @@ Here's a quick guide to create a pull request for your simplenote-android patch:
 [build-instructions]: https://github.com/Automattic/simplenote-android/blob/develop/README.md#how-to-configure
 [pr]: https://github.com/Automattic/simplenote-android/compare/
 
-# Branching
+## Branching
 
 * `develop` branch represents the cutting edge version. This is probably the one you want to fork from and base your patch on. This is the default github branch.
 * Fix or feature branches. Proposed new features and bug fixes should live in their own branch. Use the following naming convention: if a github issue exists for this feature/bugfix, the branch will be named `issue/ISSUEID-comment` where ISSUEID is the corresponding github issue id. If a github issue doesn't exist, branch will be named `feature/comment`. These branches will be merged in `develop`.
 
-[git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
\ No newline at end of file
+[git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
+
+## Testing
+
+Use the [testing checklist][testing-checklist] as a guide for smoke testing the app after making code changes. Looking for other ways to contribute? You can also use the checklist as a guide for beta testing the app.
+
+You can install a beta version in one of two ways:
+
+1. On your Android device, head to [Simplenote in the Google Play Store][play-store], scroll down to the “Join the beta” section, and tap “Join” to opt in to the beta program. You’ll get the latest beta as an update in the Play Store. (You'll continue to get updates each time a new beta is released, which is useful for ongoing beta testing.)
+2. Alternately, install the latest pre-release version from the [Releases][releases] page. (This is good for one-off testing, or beta testing without the Google Play Store on your device.)
+
```

Comment:
Ah, really good point. I'll combine that with option 1, since it seems the best option given where people will most likely be viewing the documentation.